packages/bits/src/lib/select-v2/combobox-v2/combobox-v2.component.less

```diff
@@ -71,6 +71,11 @@
             border-right: 1px solid;
             .setCssVariable(border-right-color, nui-color-line-default);
         }
+
+        &:focus {
+            border-radius: 6px;
```

Comment:
6px seems like a strange number perhaps. Was 6px suggested by UX? Also, a file-scoped variable for this at the top of the file might be good.

Reply:
We don't have spec for this focus, because of overflow styles inside combobox. I used mixin with inner shadow and make. I set 6px for radius to avoid rectangular focus. But I'll clarify it with ux.

Reply:
clarify with ux and reduce border radius to 3px